WebClient/src/main/resources/webapp-server.yml

```diff
@@ -0,0 +1,11 @@
+# Spring Properties
+spring:
+  application:
+     name: web-app  # Service registers under this name
```

Comment:
http://nooooooooooooooooooooooo.com (number of o's is subject to change). No web-app-app... :sob: